src/YQWizard.cc

```diff
@@ -774,11 +774,13 @@ QWidget *YQWizard::layoutWorkArea( QWidget * parent )
     //headingHBox->setSizePolicy( QSizePolicy( QSizePolicy::Expanding, QSizePolicy::Minimum ) ); // hor/vert
     leftInnerBox->addLayout( headingHBox );
 
+    /*
```

Comment:
Please either use `#if 0 .... #endif` or simply delete that block; it is too easily overlooked that this is commented out. This very much depends on the editor that people use and how good that editor's syntax highlighting is.

Reply:
Noted